src/cli/main.c

```diff
@@ -37,6 +37,7 @@ usage(void)
 	fprintf(stderr, "usage: %s\n"
 	    "\t-k\tConfigure new network [provisioning key]\n"
 	    "\t-l\tList networks\n"
+		"\t-L\tName network\n"
```

Comment:
I guess it could be specified that this option should be used with the `-k` option
like ./netvfy-agent -L network_name -k prov_key

Reply:
yup fixed!

Reply:
tiny nit: here the spacing is: 1 tab + 4 spaces, as per the openbsd bible :)
"Indentation is an 8 character tab. Second level indents are four spaces." https://man.openbsd.org/style

Reply:
Also regarding the question -L vs -N
I had "label" in mind, but I like the "Name" idea, which means that we could use "-n". Also when you do ./netvfy-agent -h, I guess we could print -k and then -n directly after